spec/unit/hanami/helpers/html_helper/html_builder_spec.rb

```diff
@@ -245,6 +245,13 @@
       result = @builder.input('required' => true, 'value' => 'Title "book"', 'something' => 'bar').to_s
       expect(result).to eq('<input required="required" value="Title "book"" something="bar">')
     end
+
+    # rubocop:disable Style/SymbolArray
```

Comment:
Rubocop wants us to use `%I[]` or `%i[]` here, but that makes it less obvious what's going on, if you're not familiar with that synatx. I can do that instead, here, if others think it's a good idea.